web/src/components/dialog/DialogHolder.svelte

```diff
@@ -4,6 +4,7 @@
     import SmallDialog from "$components/dialog/SmallDialog.svelte";
     import PickerDialog from "$components/dialog/PickerDialog.svelte";
     import SavingDialog from "$components/dialog/SavingDialog.svelte";
+    import NoScriptDialog from "./NoScriptDialog.svelte";
```

Comment:
this line should be above `SmallDialog`, as it's the shortest one out of components

Reply:
use $components routing here, too